src/allmydata/windows/registry.py

```diff
@@ -1,5 +1,5 @@
 import sys
-import _winreg
+import winreg
```

Comment:
I thought `winreg` was named `_winreg` in Python 2.7.  How does this work on Python 2.7 + Windows configuration?